server/models/meal.js

```diff
@@ -12,7 +12,7 @@ const mealSchema = new Schema({
     requirements: String,
     createdAt: { type: Date, default: Date.now },
     updatedAt: Date,
-    chefId: { type: Schema.Types.ObjectId, ref: "Chefs" },
+    chefId: { type: Schema.Types.ObjectId, ref: "Chef" },
```

Comment:
shouldn't it be `Chefs`

Reply:
I don't think so. I only changed it for consistency with the User and Meal models.